de.dlr.sc.virsat.model.extension.requirements.feature/.checkstyle

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<fileset-config file-format-version="1.2.0" simple-config="true" sync-formatter="false">
```

Comment:
Ok so this is where the misplaced comment from the previous review should have gone:
Features shouldnt have .checkstyle files.